packages/ember/src/index.ts

```diff
@@ -0,0 +1,43 @@
+export function setupAppsignal(appsignal: any, Ember = (window as any).Ember) {
+  const prevHandler = Ember.onerror
+
+  if (!appsignal) {
+    throw new Error("No instance of Appsignal supplied.")
+  }
+
+  if (!Ember) {
+    throw new Error("No instance of Ember found.")
+  }
+
+  Ember.onerror = function(error: Error): void {
+    const span = appsignal.createSpan((span: any) =>
+      span.setAction("Ember.onerror").setError(error)
+    )
+
+    appsignal.send(span)
+
+    if (prevHandler && typeof prevHandler === "function") {
+      prevHandler.apply(this, arguments as any)
+    } else if (Ember.testing) {
+      throw error
```

Comment:
Does this mean we only rethrow the error in testing? Why not in production?

Reply:
This is so any assertions dependent on throwing are triggered in testing mode. We don't normally rethrow in the other integrations, so this is for consistency.